docs/developer/app-store/apps/invoices.mdx

```diff
@@ -83,3 +83,22 @@ Visit the [development guide](../../extending/apps/developing-apps/app-examples)
 Apart from common env variables, the Invoice app accepts the following:
 
 - `TEMP_PDF_STORAGE_DIR=` - a temp directory where the PDF files will be stored. In Vercel, set `/tmp`. Add it to `.gitignore`
+
+## Migration from the _Invoicing_ plugin
+
+The invoices app covers the functionality of the plugin. In the future, plugins will be deprecated and replaced by apps. To migrate from plugin to app, follow the steps below:
```

Comment:
**suggestion (non-blocking)**:
link to the plugin docs

Reply:
There is no docs for this plugin